media/js/social.js

```diff
@@ -0,0 +1,32 @@
+(function(win, doc, $) {
+
+    $('.share-link').on('click', function(e) {
+        e.preventDefault();
+
+        var $this = $(this);
+        var title = encodeURIComponent($this.data('title') || doc.title);
+        var url = encodeURIComponent($this.data('url') || win.location);
```

Comment:
We should add `utm_*` tracking parameters for this so we can segment and analyze visitors who come from this feature. I suggest:
`utm_campaign=share&utm_source=[twitter.com|facebook.com|plus.google.com]&utm_medium=share widget`